apps/passport-server/src/services/multiProcessService.ts

```diff
@@ -78,15 +130,23 @@ export class MultiProcessService {
     // to the worker farm in a promise.
     return traced(LOG_NAME, "verifySignaturePCD", async () => {
       logger("[MULTIPROCESS] verifying a semaphore signature");
-      return new Promise<boolean>((resolve, reject) => {
-        this.workers(JSON.stringify(pcd), (err: Error, result: boolean) => {
+      const payload = JSON.stringify(pcd);
```

Comment:
i dont think multiprocess service is the right place for caching, it feels like the sort of thing that should live outside of it. for now thought I think this is fine to have this way.